scripts/codelists.json

```diff
@@ -33,3 +33,4 @@
 {"_id":"opss-scheme-membership-sizeband","name":"schememembershipsizeband","links":{"self":{"id":"opss-scheme-membership-sizeband","href":"http://localhost:22400/code-lists/opss-scheme-membership-sizeband"},"codes":{"href":"http://localhost:22400/code-lists/opss-scheme-membership-sizeband/codes"}}}
 {"_id":"opss-status","name":"status","links":{"self":{"id":"opss-status","href":"http://localhost:22400/code-lists/opss-status"},"codes":{"href":"http://localhost:22400/code-lists/opss-status/codes"}}}
 {"_id":"wellbeing-geography","name":"geography","links":{"self":{"id":"wellbeing-geography","href":"http://localhost:22400/code-lists/wellbeing-geography"},"codes":{"href":"http://localhost:22400/code-lists/wellbeing-geography/codes"}}}
+{"_id":"mmm-yy","Name":"time","Links":{"self":{"id":"mmm-yy","href":"http://localhost:22400/code-lists/mmm-yy"},"codes":{"href":"http://localhost:22400/code-lists/mmm-yy/codes"}}}
```

Comment:
the fields Name and Links need to be lower cased

Reply:
Fixed